src/scripts/db_script.sql

```diff
@@ -0,0 +1,23 @@
+-- DATABASE : ea_restaurant_db
+
+CREATE DATABASE ea_restaurant_db
+
+-- Enum : status
+
+CREATE TYPE status AS ENUM ('ACTIVE', 'DELETE');
+
+-- Table : products
+
+CREATE TABLE IF NOT EXISTS products
+									(
+    									product_id serial NOT NULL ,
```

Comment:
Usa capital letters for all postgres keywords. For instance rename this to: SERIAL